client/src/modules/stock/lots/registry.js

```diff
@@ -18,6 +18,7 @@ function StockLotsController(Stock, Notify,
   var vm = this;
 
   var cacheKey = 'lot-grid';
+  var filterKey = 'lot';
```

Comment:
Instead of defining a `filterKey` like this, can you just do:
```js
var StockLotFilters = Stock.filter.lot;

// now call StockLotFilters.method()
```

This would make it really easy to read - at the moment, every time you use it I wonder whether `Stock.filters[filterKey]` is really defined, or an object, or ...